<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-63VPLUC" failures="1" tests="1" name="Test(failed)" time="30.952" errors="0" timestamp="2024-03-04T14:46:17 IST">
  <testcase classname="listeners.Listenerspractice" name="ts_01" time="0.044">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at listeners.Listenerspractice.ts_01(Listenerspractice.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- ts_01 -->
  <testcase classname="listeners.Listenerspractice" name="ts_02" time="0.0">
    <ignored/>
  </testcase> <!-- ts_02 -->
</testsuite> <!-- Test(failed) -->
